wdn/templates_5.0/scss/deprecated/deprecated.tables.scss

```diff
@@ -17,7 +17,7 @@ caption {
 
 thead,
 tbody {
-  @include bb;
+  //@include bb;
```

Comment:
Can these commented out lines be removed?

Reply:
They are commented out because @skoolbus39 is making some changes to DCF and SASS couldn't find it in my version of DCF.. @skoolbus39, thoughts?

Reply:
I can make sure they get updated. Would you prefer I pull down this branch and update the version of DCF used, then make the changes?